test/unit/server/services/procedures/icesheets.spec.js

```diff
@@ -0,0 +1,12 @@
+describe('icesheets', function() {
+    const utils = require('../../../../assets/utils');
+
+    utils.verifyRPCInterfaces('EarthOrbit', [
```

Comment:
This should be failing currently. Is it not? (`'EarthOrbit'` should be update to the IceSheets service name.)